TestApp/ios/Podfile

```diff
@@ -7,9 +7,9 @@ target 'TestApp' do
   # use_frameworks!
 
   # Pods for TestApp
-  pod 'AppCenter/Crashes', '~> 1.9.0'
-  pod 'AppCenter/Analytics', '~> 1.9.0'
-  pod 'AppCenter/Push', '~> 1.9.0'
+  pod 'AppCenter/Crashes', '~> 1.9.1-3'
```

Comment:
Temporary version until Native SDK released, revert that after native SDK released.